src/advanced/bitcoin-anchoring.md

```diff
@@ -509,7 +509,9 @@ Requires [observer interval](#observer-interval) to be set.
 
 Returns the content of the anchoring transaction which anchors the
 specific block. If the selected block has not been anchored yet or if the
-[observer interval](#observer-interval) is not set, returns `null`.
+[observer interval](#observer-interval) is not set,
+or number of confirmations of anchoring transaction is less than
+[utxo confirmations][anchoring-parameters] parameter, returns null.
```

Comment:
> or number of confirmations of anchoring transaction is less than [utxo confirmations][anchoring-parameters] parameter, returns null.

or if the number of confirmations of anchoring transaction is less than the [utxo confirmations][anchoring-parameters] parameter, returns null.